onboarding/src/Components/LeftMenu.js

```diff
@@ -19,9 +19,13 @@ const LeftMenu = ({ packagesList }) => {
         : setShowFolders(false);
   }, [location.pathname]);
 
+  const handleHideAside = () => {
+    setToggleAside(false);
+  }
+
   return (
-    <aside className="app-aside app-aside-expand-md app-aside-light">
-      <div className="aside-content">
+    <aside className={`app-aside app-aside-expand-md app-aside-light ${showAside ? "show" : ""}`} style={{zIndex: "9"}}>
```

Comment:
Without `style={{zIndex: "9"}}`, 'x' button to close the menu didn't appear.
Value `1` was enough (`style={{zIndex: "1"}}`
but we can stick with `9` :slightly_smiling_face: